src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.Analyzer.cs

```diff
@@ -36,53 +36,6 @@ public static AnalyzerResult Analyze(SelectionResult selectionResult, bool local
                 return CreateFromSymbolCommon<LocalDeclarationStatementSyntax>(compilation, symbol, type, style, s_nonNoisySyntaxKindSet);
             }
 
-            protected override int GetIndexOfVariableInfoToUseAsReturnValue(IList<VariableInfo> variableInfo)
```

Comment:
moved to base type.  the one difference between VB and C# is controlled by the `bool treatoutasref` flag above.